sdk/OoyalaSkinSDK/controlBar.js

```diff
@@ -60,7 +60,11 @@ var ControlBar = React.createClass({
     if (this.props.live) {
       return this.props.live.label;
     } else {
-      return Utils.secondsToString(this.props.playhead) + "/" + Utils.secondsToString(this.props.duration);
+      var durationString = Utils.secondsToString(this.props.playhead);
```

Comment:
hmm maybe not.  as long as you can tell me this is done in the same way "adBar" is done